bindings/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+cmake_minimum_required(VERSION 3.12)
+cmake_policy(VERSION 3.12)
+
+project(PortAudio LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED True)
+
+# Todo (multi-generator): Add support for multiple generators like:
+# - {Debug, Release} x {Static, Dynamic} x {MT, MD (Windows only)}
+set(portaudio-cpp-sources)
+add_subdirectory(source/portaudiocpp) # define source-files
+
+add_library(portaudiocpp ${portaudio-cpp-sources})
+add_library(PortAudio::portaudiocpp ALIAS portaudiocpp) # For subdirectory build
+
+# Remove, when PortAudio::portaudio-cpp has it's own CMakeLists.txt
```

Comment:
This comment is confusing. This is portaudio-cpp's CMakeLists.txt, so why is this still here? Do you mean remove this when the C++ bindings are moved to another repository?

Reply:
Yep, forgot that line.